src/java.net.http/share/classes/jdk/internal/net/http/Http1Response.java

```diff
@@ -416,6 +421,10 @@ private void onFinished() {
         }
     }
 
+    public void closeWhenFinished() {
```

Comment:
Hello Conor, do you think it might be better if we make this package private access instead of `public`?

Reply:
Yes - good catch!

Reply:
Good suggestion, no need for it to be public. Will do.

Reply:
I made this change but added protected by mistake instead of just removing the modifier. Will fix in subsequent change